app/controllers/game_controller.rb

```diff
@@ -22,7 +24,8 @@ def turn
     session['current_player'] = @players.current_player
 
     respond_to do |format|
-      format.json { render json: { board: @board, players: @players, current_player: @players.current, finished: @finished } }
+      format.json { render json: { board: @board, players: @players,
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.